consensus/src/block_storage/block_store_test.rs

```diff
@@ -396,41 +396,3 @@ fn test_need_fetch_for_qc() {
         NeedFetchResult::QCAlreadyExist,
     );
 }
-
-#[test]
-fn test_empty_reconfiguration_suffix() {
```

Comment:
@zekun000 for you to review

Reply:
do we have counter-part in LEC test?

Reply:
LEC has no way to know the reconfig logic. I think the test self is sort of hacky since the blocks in test are inserted manually. The blocker is MockStateComputor.

Reply:
I thought we move the hack from block store to LEC? so that LEC now provides the guarantee any block has a parent with reconfiguration has the same execution result.

Reply:
Yes. But the test here definitely belongs to consensus. LEC doesn't know what reconfiguration flow is. The problem is the way how this test runs is hacky two with `TreeInserter`. If you really want these tests let me see whether I can hack into it.

Reply:
LEC can test executing a reconfiguration block and then executing a child block and assert the result is the same?

the empty part belongs to consensus

Reply:
This part is easier. Then I will leave the empty part for you later :)

Reply:
@zekun000 plz take a look on the new test I added.

Reply:
you can keep this test but just delete a5?

Reply:
a6 is the child of a5...

Reply:
status here?